scripts/configure_invokeai.py

```diff
@@ -587,31 +557,63 @@ def get_root(root:str=None)->str:
             match = re.search('--root\s*=?\s*"?([^"]+)"?',l)
             if match:
                 root = match.groups()[0]
-    root = root or '.'
-    return root.strip()
+                root = root.strip()
+    return root
+
+#-------------------------------------
+def select_root(yes_to_all:bool=False):
+    default = os.path.expanduser('~/invokeai')
+    if (yes_to_all):
+        return default
+    completer.set_default_dir(default)
+    completer.complete_extensions(())
+    completer.set_line(default)
+    return input(f"Select a directory in which to install InvokeAI's models and configuration files [{default}]: ")
+
+#-------------------------------------
+def select_outputs(root:str,yes_to_all:bool=False):
+    default = os.path.normpath(os.path.join(root,'outputs'))
+    if (yes_to_all):
+        return default
+    completer.set_default_dir(os.path.expanduser('~'))
+    completer.complete_extensions(())
+    completer.set_line(default)
+    return input('Select the default directory for image outputs [{default}]: ')
```

Comment:
If `Globals.root` is `''` (empty string), and you accept the defaults, the end result is that, inside `initialize_rootdir()`, `root` and `outputs` are also empty strings. We need to return the sensible defaults.

```suggestion
#-------------------------------------
def select_root(yes_to_all:bool=False):
    default = os.path.expanduser('~/invokeai')
    if (yes_to_all):
        return default
    completer.set_default_dir(default)
    completer.complete_extensions(())
    completer.set_line(default)
    selected_directory = input(f"Select a directory in which to install InvokeAI's models and configuration files [{default}]: ")
    return selected_directory or default

#-------------------------------------
def select_outputs(root:str,yes_to_all:bool=False):
    default = os.path.normpath(os.path.join(root,'outputs'))
    if (yes_to_all):
        return default
    completer.set_default_dir(os.path.expanduser('~'))
    completer.complete_extensions(())
    completer.set_line(default)
    selected_directory = input(f'Select the default directory for image outputs [{default}]: ')
    return selected_directory or default
```

Reply:
Responses:
1. I've changed the logic a bit. When the user provides a relative path for root, including the empty string and '.'. The script converts the path into an abspath and configures that as the root. So installs to '' and '.' will both go into the repo, which is consistent with previous behavior.
2. Fixed
3. Fixed